p2p/proto/receipt.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+import "p2p/proto/common.proto";
+
+message MessageToL1 {
+  Felt252 from_address = 1;
+  repeated Felt252 payload = 2;
+  EthereumAddress to_address = 3;
+}
+
+message EthereumAddress { bytes elements = 1; }
+
+message MessageToL2 {
+  EthereumAddress from_address = 1;
+  repeated Felt252 payload = 2;
+  Felt252 to_address = 3;
+  Felt252 entry_point_selector = 4;
+  Felt252 nonce = 5;
+}
+
+message CommonTransactionReceiptProperties {
+  Felt252 transaction_hash = 1;
+  uint32 transaction_index = 2;
+  Felt252 actual_fee = 3;
+  repeated MessageToL1 messages_sent = 4;
+  // repeated Event events = 5; separate
+  // Optional
+  MessageToL2 consumed_message = 5;
+  ExecutionResources execution_resources = 6;
+}
+
+message ExecutionResources {
+  message BuiltinInstanceCounter {
+    uint64 bitwise_builtin = 1;
+    uint64 ecdsa_builtin = 2;
+    uint64 ec_op_builtin = 3;
+    uint64 output_builtin = 4;
+    uint64 pedersen_builtin = 5;
+    uint64 range_check_builtin = 6;
```

Comment:
Missing the poiseidon and with v0.12.1 the keccak builtins.

https://github.com/starkware-libs/cairo-lang/blob/cc86dfc40202fd74370734ef8b9414a2d78dcc06/src/starkware/cairo/lang/builtins/all_builtins.py#L52-L59